website/app/styles/doc-components/layout.scss

```diff
@@ -0,0 +1,27 @@
+.doc-layout {
+  display: flex;
+  flex-flow: wrap;
+  gap: var(--doc-layout-spacing);
+  outline: 3px dotted blue;
+}
+
+// Direction:
+
+.doc-layout--direction-horizontal {
+  flex-direction: row; // this is the browser default
+}
+
+.doc-layout--direction-vertical {
+  // flex-direction: column;
+
+  // Experimental: make "align" values (left, right, center) have a consistent effect...
+  // (forces each child item to occupy its own row)
+  column-gap: 100%;
```

Comment:
This is what I came up with so that styles and naming for alignment options could be simplified and made consistent. (If you change the flex-direction to column instead then the align styles would have a different effect.)

Reply:
Can you expand why using the `flex-direction` would not work? I don't understand

Reply:
It works but it would require different additional styles for setting the alignment of the content. I wanted to simplify the styles for alignment and make them consistent. (And I didn't want the API of the Layout component to be too complicated or replicate underlying flexbox styles)

I created a demo page: https://codepen.io/KristinB/pen/LYBzJbJ/7fa025e1e32ca066e056cf104bd8bc4f